AdobeStockImageAdminUi/view/adminhtml/web/js/mediaGallery/grid/columns/image/licenseActions.js

```diff
@@ -98,11 +98,13 @@ define([
                     this.imageModel().addMessage('success', $.mage.__('The image has been licensed.'));
                 }.bind(this)).fail(function (error) {
                     if (error) {
-                        this.imageModel().addMessage('error', error);
```

Comment:
Can we refactor this method to be a "deferred" "promise" ? and catch any error ? and then display it in our grid messages ?

Reply:
Good point @Nazar65 . Yes, I think it is possible but in the case of the Sign In the error is shown in the Adobe Stock Sign In window and the text we get from it describes only Authentification window was closed.To show the same error message twice think not so useful. @chalov-anton 

This is the signing window with an error message. Looks informative.
![image](https://user-images.githubusercontent.com/5318512/86023180-00a37700-ba34-11ea-88a2-aa5f005e74b6.png)

Reply:
@coderimus Agree, the text in the message looks informative enough.